WordPress/JetpackScreenshotGeneration/JetpackScreenshotGeneration.swift

```diff
@@ -95,3 +96,16 @@ extension BaseScreen {
         return self
     }
 }
+
+extension ScreenObject {
+
+    @discardableResult
+    func thenTakeScreenshot(_ index: Int, named title: String) -> Self {
+        let mode = XCUIDevice.inDarkMode ? "dark" : "light"
+        let filename = "\(index)-\(mode)-\(title)"
+
+        snapshot(filename)
+
+        return self
+    }
+}
```

Comment:
This is a 1:1 copy of the extension on `BaseScreen` above and is a way to combine Fastlane's screenshot logic with a nice DSL in `ScreenObject`.

Eventually, we might want to add some kind of abstraction and make this part of the framework. For the moment, though, the focus is to convert all the screens, so this will do.